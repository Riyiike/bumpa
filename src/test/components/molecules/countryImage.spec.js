import React from "react";
import { shallow } from 'enzyme'
import BorderCountries from "../../../components/molecules/borderCountries";



// 

describe('test mainPage', () => {
    it('test mainPage', () => {
        const btn = shallow(<BorderCountries />)
        expect(btn).toMatchSnapshot();
    })
})