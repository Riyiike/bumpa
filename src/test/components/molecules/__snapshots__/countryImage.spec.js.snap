// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test mainPage test mainPage 1`] = `""`;
